src/Http/Http.Extensions/src/RequestHeaders.cs

```diff
@@ -279,11 +279,26 @@ public Uri Referer
             }
         }
 
+        /// <summary>
+        /// Gets the value of header with <paramref name="name"/>.
+        /// </summary>
+        /// <typeparam name="T">The type of the header.
+        /// The given type must have a static TryParse method</typeparam>
+        /// <param name="name">The name of the header to retrieve.</param>
+        /// <returns>The value of the header</returns>
         public T Get<T>(string name)
         {
             return Headers.Get<T>(name);
         }
 
+        /// <summary>
+        /// Gets the values of header with <paramref name="name"/>.
+        /// </summary>
+        /// <remarks><typeparamref name="T"/> must contain a TryParseList method with the signature <code>public static bool TryParseList(IList&lt;string&gt;, out IList&lt;T&gt;)</code></remarks>
```

Comment:
You used `typeparamref` here but not in the `Get` method. I'm fine with doing this either way but we should be consistent.

Similarly, you put the full signature in here, but not above.

Reply:
We should also be consistent between `remarks` and `typeparam` items. Let's make them similar in format and consistent between both methods.